pandas/io/formats/style.py

```diff
@@ -257,8 +274,8 @@ def format_attr(pair):
                 row_es.append(es)
             head.append(row_es)
 
-        if self.data.index.names and not all(x is None
-                                             for x in self.data.index.names):
+        if self.data.index.names and index_visible\
```

Comment:
Could you write this is

```python 
    if (self.data.index.names and index_visible
            and not all(x is None for x in self.data.index.names)):
```

we try to avoid continuation backslashes